src/client/styles/styles.less

```diff
@@ -1947,3 +1947,12 @@ a {
 }
 
 /*#endregion */
+
+/*Dashboard*/
+.dashboard {
+  border-radius: 30px; background: rgb(255,255,255);
+  background: -moz-linear-gradient(-45deg,  rgba(255,255,255,1) 0%, rgba(206,206,206,1) 13%, rgba(181,181,181,1) 54%, rgba(209,209,209,1) 70%, rgba(214,214,214,1) 87%, rgba(246,246,246,1) 100%);
```

Comment:
the main thing is to know that in less files you should not write vendor prefixes. Recipes are here http://stackoverflow.com/questions/18558368/is-there-a-generic-way-to-add-vendor-prefixes-in-less 

But I don't require you to do that. Just note how it can be done.